test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Get_ApplicationJson_DataType_HttpTrigger.cs

```diff
@@ -0,0 +1,29 @@
+using System.Net;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp.Models;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp
+{
+    public static class Get_ApplicationJson_DataType_HttpTrigger
+    {
+        [FunctionName(nameof(Get_ApplicationJson_DataType_HttpTrigger))]
+        [OpenApiOperation(operationId: nameof(Get_ApplicationJson_DataType_HttpTrigger.Get_ApplicationJson_DataType), tags: new[] { "greeting" })]
```

Comment:
Can you change the tag from `greeting` to `dataType`?

Reply:
Oh, I missed this part. I modified it.